cli/cmd/usergroup/create.go

```diff
@@ -23,9 +23,11 @@ func CreateCmd(ch *cmdutil.Helper) *cobra.Command {
 				name = args[0]
 			}
 
-			err = cmdutil.StringPromptIfEmpty(&name, "Enter user group name")
-			if err != nil {
-				return err
+			if name == "" {
+				err = cmdutil.StringPromptIfEmpty(&name, "Enter user group name")
+				if err != nil {
+					return err
+				}
```

Comment:
Same comment as in the previous review – I think `StringPromptIfEmpty` already checks this